### PR TITLE
DHFPROD-8228: Make Hub Central Documentation Links Dynamic

### PR DESCRIPTION
**Pull request by briantang**
### Description

- Duplicate PR for 5.6

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Review by AkshayBajajML — APPROVED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@briantang Clicking on documentation links navigates me to 5.7, Instead it should navigate to 5.6 right, since this is a 5.6 branch, Could you please look into it ?

**Comment by briantang** [comment]
@Sanjeevani19 Since the version is coming from the environment sent by the backend, you need to do a rebuild on the 5.6 branch and it will direct you to the correct docs. Just confirmed that this works after rebuilding on the 5.6 branch.

**Comment by Sanjeevani19** [comment]
> @Sanjeevani19 Since the version is coming from the environment sent by the backend, you need to do a rebuild on the 5.6 branch and it will direct you to the correct docs. Just confirmed that this works after rebuilding on the 5.6 branch.

Thanks for confirming that, Is it possible to add any test cases for this scenario ? I am sure it will most likely a URL change, but any test you could come up with

File: marklogic-data-hub-central/ui/src/pages/Overview.test.tsx [hunk 1]
```diff
@@ -164,7 +166,7 @@ describe("Overview component", () => {
     expect(documentationLinks.length === enabled.length); // All cards should have Documentation links
     documentationLinks.forEach((docLink, i) => {
       fireEvent.click(docLink);
-      expect(mockedWindowOpen).toBeCalledWith(overviewConfig.documentationLinks[enabled[i]], "_blank");
+      expect(mockedWindowOpen).toBeCalledWith(overviewConfig.documentationLinks.tileSpecificLink(5.3, enabled[i]), "_blank");
```
Comment (briantang) [inline marklogic-data-hub-central/ui/src/pages/Overview.test.tsx:169]: Thanks @Sanjeevani19! In this line, I believe I have verified that when the environment data is set to 5.3, the mock window is opened with the "5.3" version number.

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**